carbonmark/components/pages/Retire/Activity/Overview.tsx

```diff
@@ -34,14 +32,25 @@ export const RetireOverview: FC = () => {
 
       try {
         setIsLoadingTotals(true);
+        const klimaRetires = await queryKlimaRetiresByAddress(address);
 
-        const [totalRetirements, totalCarbonRetired] = await Promise.all([
-          getTotalRetirements({ beneficiaryAddress: address }),
-          getTotalCarbonRetired({ beneficiaryAddress: address }),
-        ]);
+        const totalRetiredTonnesWei = klimaRetires.reduce((acc, r) => {
+          const amount = r.retire.amount;
 
-        setTotalRetirements(totalRetirements);
-        setTotalCarbonRetired(totalCarbonRetired);
+          if (r.retire.credit.project.registry === "ICR") {
```

Comment:
it might be better to do this "edge" case formatting inside the `queryKlimaRetiresByAddress` function before we move all this in to the API